docs/05-concepts/06-database/01-connection.md

```diff
@@ -62,3 +70,59 @@ To remove the database and __delete__ all data add the `-v` flag.
 $ docker compose down -v
 ```
 
+
+## Connecting to a Postgresql Server Instance
+
+Connecting to a Postgresql Server Instance is similar to connecting to a Docker Image, but there are some prerequisites to take into account:
+
+- Making sure that the machine/server allows incoming traffic
+- Creating a Postgresql user and database
```

Comment:
Suggestion: Add instructions for how to create a user and a database.

This is the step that many people get stuck on, I think it would be good to have instructions on creating a database with a user with the right permissions.

That obsticle that Craig Labenz was fighting with in [observable flutter](https://youtu.be/U4iMFNEyyk0?list=PLjxrf2q8roU1GHtc2FCHoEZr_v-LqnTZX&t=1041)

Reply:
I would recommend watching that part of the video and ensuring we answer all those questions.